### PR TITLE
VACMS-4855: Configuration for Sections dashboard based on Product vocabulary.

### PR DESCRIPTION
**Pull request by kevwalsh**
## Description

See #4850. 

## Testing done

Manual, behat. 

## QA steps

Validate that the view mode and changes to views have not caused a regression for CMS users.
- [x] As a content_admin or admin, visit a variety of Section taxonomy term pages and ensure that the content matches what's on prod. (eg no changes)

Validate that the removal of fields from the Sections vocabulary have not caused a regression on va.gov
- [ ] Build va.gov and visit a sampling of Benefits Hub landing pages (/health-care, /disability, etc... any of the 11 listed on the homepage). The right rail content under "Connect with us" should remain identical to va.gov prod

Edit a Section taxonomy term such as term 77 or 5
- [x] See the optional Product field with a list of Products 
- [x] See that the form no longer contains useless fields 

Edit_term___VA_gov_CMS

Validate the Vet Center Dashboard view form mode is configured with the Vet Center facility listing
- [x] Go to /admin/structure/taxonomy/manage/administration/overview/display/vet_center_dashboard/layout and validate the placement of the block.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: tests/behat/drupal-spec-tool/views.feature [hunk 1]
```diff
@@ -159,7 +159,7 @@ Feature: Views
 | Search | Master | default | Master |
 | Search | Page | page | Page |
 | Taxonomy term | Master | default | Master |
-| Taxonomy term | Feed | feed_1 | Feed |
```
Comment (kevwalsh) [inline tests/behat/drupal-spec-tool/views.feature:162]: Removed a useless display for this view.

File: config/sync/field.storage.taxonomy_term.field_metatags.yml [hunk 2]
```diff
@@ -1,19 +0,0 @@
-uuid: acaea178-c05b-4531-82de-2f5a1c0b2bc8
-langcode: en
-status: true
-dependencies:
-  module:
-    - metatag
```
Comment (kevwalsh) [inline config/sync/field.storage.taxonomy_term.field_metatags.yml:6]: Metatag has no purpose on a vocabulary that has no FE representation.

File: config/sync/views.view.taxonomy_term.yml [hunk 3]
```diff
@@ -1516,47 +1516,6 @@ display:
       tags:
         - 'config:field.storage.node.field_administration'
         - 'config:workflow_list'
-  feed_1:
```
Comment (kevwalsh) [inline config/sync/views.view.taxonomy_term.yml:1519]: Removing a view display useless in a decoupled site.

File: config/sync/field.field.taxonomy_term.administration.field_intro_text.yml [hunk 4]
```diff
@@ -1,19 +0,0 @@
-uuid: ac9a18be-2025-4cbf-85a5-3b75aa823a67
-langcode: en
-status: true
-dependencies:
-  config:
-    - field.storage.taxonomy_term.field_intro_text
-    - taxonomy.vocabulary.administration
-id: taxonomy_term.administration.field_intro_text
```
Comment (kevwalsh) [inline config/sync/field.field.taxonomy_term.administration.field_intro_text.yml:8]: This field was not in use in the FE.

File: config/sync/field.storage.taxonomy_term.field_acronym.yml [hunk 5]
```diff
@@ -1,21 +0,0 @@
-uuid: 81bced68-2b87-4d22-8966-122538e53748
```
Comment (kevwalsh) [inline config/sync/field.storage.taxonomy_term.field_acronym.yml:1]: Deleting this field after validating that there is are only 5 values, it's not used anywhere for any business or FE reason.  

section_description_and_acronym__Taxonomy_term____VA_gov_CMS

**Comment by ethanteague** [comment]
Vets-web pr that will fix graphql build errors: https://github.com/department-of-veterans-affairs/vets-website/pull/16673

**Comment by swirtSJW** [comment]
This is blocked on this PR https://github.com/department-of-veterans-affairs/vets-website/pull/16673 AND a PR on CMS side to bring in the new vetsweb into CMS.

**Comment by swirtSJW** [comment]
This is now in our codebase https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4969 so I think we should be able to get passing tests now.  Rebasing to trigger tests.

**Pull request closed without merging**